composer.json

```diff
@@ -27,6 +27,15 @@
 		}
 	},
 
+	"scripts": {
+		"phpunit": "vendor/bin/phpunit --configuration test/phpunit/phpunit.xml",
+		"phpstan": "vendor/bin/phpstan analyse --level 6 src",
```

Comment:
By the way, PHPStan could easily be increased to a higher level, but that is best done in another PR.
Likewise, the CI could also run at a higher level (or use a `.phpstan.dist.neon` file to avoid having to update multiple locations)